File: NETMouse - .NET release/Examples/PascalABC.NET/Arrays_12.pas
{$reference ABCNET.dll}
uses ABCNET.Utils;
uses ABCNET.Extensions;
begin
  var c := BaseU.ReadInteger('C:');
  Arr.ReadInteger(10, 'Элемент {0}-ый:').Where(x -> x > c).Average().Println();
  BaseU.ReadChar();
end.

File: NETMouse - .NET release/Examples/C#/Arrays_12.cs
using ABCNET.Extensions;
using ABCNET.Utils;
using System.Linq;

namespace TestProgram
{
    internal class Program
    {
        private static void Main(string[] args)
        {
            int c = Base.ReadInteger("C:");
            Arr.ReadInteger(10, "Элемент {0}-ый:").Where(x => x > c).Average().Println();
        }
    }
}